### DISCUSSION TITLE
v-viewer on Vue 3

### DISCUSSION THREAD

**Discussion by FCardeira** [mirari/v-viewer#174] [Q&A]
Hi,

Are you planning on releasing a version of v-viewer for vue 3?

Best Regards

**Comment by mirari** [comment]
Under refactoring and will be released in a few days, thank you for your attention!

**Comment by mirari** [comment]
https://github.com/mirari/v-viewer/tree/v3
Vue3 version has been released, welcome to use and feedback, thanks for your support.
